### ISSUE TITLE
Indenting child properties in Mouse Preferences

### ISSUE THREAD

**Issue by philipzae** [mate-desktop/mate-control-center#228]
Some items in the touchpad tab of Mouse Preferences arent indented, so it would be good to do so. Below is a mockup of what it could look like if things are indented and aligned correctly.

![touchpad tab mockup](https://cloud.githubusercontent.com/assets/4412114/13381638/211ed668-de7a-11e5-9ba8-5a173b8da94f.png)

**Comment by tamplan** [comment]
Yey fixed?
![Capture du 2019-06-09 14-11-46](https://user-images.githubusercontent.com/50292710/59158818-a3ebbc00-8ac0-11e9-8518-042a8ca553b1.png)

**Comment by philipzae** [comment]
doesnt look fixed to me.

**Comment by tamplan** [comment]
Sorry, just seen the space 'Two-finger...'.

Note the changes for the part about 'Scrolling' that is different than your screen capture.

**Comment by philipzae** [comment]
Yes the scrolling section is resolved now, as it all checkboxes now. Sad that they remove the easy ability to completely disable scrolling, as now you have to click quite a few checkboxes to turn it off.

Linux Mint 19.1 Mate
![Screenshot at 2019-06-30 20-48-03](https://user-images.githubusercontent.com/4412114/60402123-4ff26580-9b9c-11e9-9b46-80140458599b.png)
